xml/art_migration.xml

```diff
@@ -0,0 +1,692 @@
+     </para>
+   <screen>- name: cassandra
+            size: (whatever size was used originally by vertica partition, less 1%)
+            mount: /var/cassandra/data
+            fstype: ext4
+            mkfs-opts: -O large_file
+            consumer:
+              name: vertica
+          - name: cassandra_log
+            size: 1%
+            mount: /var/cassandra/commitlog
+            fstype: ext4
+            mkfs-opts: -O large_file
+   </screen>
+    </listitem>
+    <listitem>
+     <para>
+      Edit
+      <filename>~/helion/my_cloud/definition/data/control_plane.yml</filename>
+      and identify the cluster definition that already includes an entry for
+      <filename>mysql</filename> in its service-components.
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      Ensure that <systemitem>lifecycle-manager</systemitem> is included in the same
+      cluster's service-components list. If it was necessary to add this as a
+      new entry, it will also need to be removed from its original position in
+      the deployer.
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      Add a new entry for <systemitem>nova-placement-api</systemitem> alongside whichever cluster
+      already includes <systemitem>nova-api</systemitem>.
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      Commit your changes to the git model:
+     </para>
+     <screen>(deployer)> git commit -a -m "Prepare for HOS 5.1 installation"</screen>
+    </listitem>
+   </itemizedlist>
+  </listitem>
+  <listitem>
+   <para>
+    Consider the effect of control plane downtime on existing DHCP leases:
+   </para>
+   <itemizedlist>
+    <listitem>
+     <para>
+      Default DHCP lease time is 48 hours as defined in the
+      <filename>~/helion/hos/ansible/roles/neutron-common/templates/neutron.conf.j2</filename> file by the
+      <systemitem>dhcp_lease_duration</systemitem> parameter.
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      DHCP leases are administered by &o_netw; and dnsmasq. This functionality will not be available
+      during the control plane downtime.
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      DHCP clients typically renew their leases either 50% of the way into the lease time or upon
+      reboot.
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      Normally, this leaves a 24-hour window for control
+      plane downtime at any point before workload's DHCP leases run out and
+      cannot be renewed. This is enough for performing a migration.
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      To view the upcoming timeouts of DHCP leases, run the following command from a &contrnode;:
+     </para>
+     <screen>for i in $(cat /var/run/neutron/dhcp/*/leases | awk '{print $1}'); do date -d @$i; done</screen>
+    </listitem>
+    <listitem>
+     <para>
+      If you find it necessary to increase the DHCP lease time before
```

Comment:
s/you find it/it is/ -- "if you find it" can be interpreted as a bit condescending and is unnecessarily long